packages/core/integration-tests/test/server.js

```diff
@@ -2,24 +2,18 @@ const assert = require('assert');
 const path = require('path');
 const fs = require('@parcel/fs');
 const {bundler} = require('./utils');
+const {sleep} = require('@parcel/test-utils');
 const http = require('http');
 const https = require('https');
+const getPort = require('get-port');
 
-describe.skip('server', function() {
-  let server;
-  afterEach(function() {
-    if (server) {
-      server.close();
-      server = null;
-    }
-  });
-
-  function get(file, client = http) {
+describe.only('server', function() {
```

Comment:
We should probably figure out a way to stop everything after a test.

Currently I'm not sure how to stop all the servers. Perhaps we need to add an onStop event to the reporters?
It will probably be usefull for more than just tests as people will probably want to stop the bundler through the api